test/model_tests/appmodels.test.js

```diff
@@ -6,7 +6,7 @@ chai.use(chaiAsPromised);
 const expect = chai.expect;
 
 describe('app models', () => {
-  const BASE_ITEM_COUNT = 1; /** Default items key */
+  const BASE_ITEM_COUNT = 2; /** Default items key, user preferences */
```

Comment:
I don't remember where, but I started doing this in the style of

```
 const BASE_ITEM_COUNT = ['default-items-key', 'user-preferences'].length;
```

instead to make it self-documenting. Not saying this is necessary here but if we feel the need to refactor this that might be a good way to go.